src/ctapipe/image/muon/intensity_fitter.py

```diff
@@ -474,11 +474,14 @@ class MuonIntensityFitter(TelescopeComponent):
     ).tag(config=True)
 
     hole_radius_m = FloatTelescopeParameter(
-        help="Hole radius of the reflector in m",
+        help="The radius of the hole in the center of the primary mirror dish in meters."
+        "The hole has hexagonal shape (same as a single mirror facet)."
+        "It is defined with the flat-to-flat distance (LST: 1.51 m, MST: 1.2 m, SST: 0.78 m)."
+        "We approximate the hexagonal hole with a circle that has the same surface area.",
         default_value=[
-            ("type", "LST_*", 0.308),
-            ("type", "MST_*", 0.244),
-            ("type", "SST_1M_*", 0.130),
+            ("type", "LST_*", 0.74),
+            ("type", "MST_*", 0.59),
+            ("type", "SST_*", 0.38),
```

Comment:
Please do not remove the `1M`, the whole radii for the different SSTs are not the same (and the 2M telescopes currently are not even supported).

Reply:
hmm... @burmist-git claimed they are the same... If not, of course 1M should be added.